rooster.py

```diff
@@ -80,7 +77,17 @@ def can_speak(ctx):
     '''
     server = ctx.message.server
     await bot.remove_roles(user, discord.utils.get(server.roles, name='Time-OUT'))
-    
+
+@bot.command(pass_context=True, description="Autism removal. x messages at a time")
+@commands.has_any_role("Director","OG")
```

Comment:
I'm not really keen on mass deletion tbh.  i'd rather potty train the puppy rather than cover the mess with a paper towel.